CHANGELOG.md

```diff
@@ -1135,12 +1135,91 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fix missing CodeMirror styles on Firefox and Safari.
 - Fix Safari crashes in `<mwc-tab>` code.
 
-## [0.1.0-pre.2] - 2020-09-12
+## 0.1.0-pre.2 - 2020-09-12
 
 ### Fixed
 
 - Fix extra/missing files.
 
-## [0.1.0-pre.1] - 2020-09-12
+## 0.1.0-pre.1 - 2020-09-12
 
 - Initial release.
+
```

Comment:
Nit: Could you please add the [unreleased] section per keepachangelog?

E.g.

`[unreleased]: https://github.com/google/playground-elements/compare/v0.18.0...HEAD`

Reply:
Done!